### PR TITLE
Fixed Documentation

### PR DESCRIPTION
**Pull request by aaishikasb**
Fixes #...

### Changes proposed

- Fixed the Documentation (Requires Approval)
- Fixed Grammatical Errors
- Fixed Typo and Punctuation Errors

### Checklist

- [x] Includes tests covering the new functionality?
- [x] Ready for review
- [x] Follows CONTRIBUTING rules

### Notify

@markyjackson-taulia

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,28 +1,29 @@
...
 
-`COLLECTING_METRICS_PERIOD_IN_SECONDS` Async task period in seconds (Default: `120` seconds)
+`COLLECTING_METRICS_PERIOD_IN_SECONDS` Async task period in seconds (Default: `120` seconds).
 
 ## Building
 
-    mvn clean install
-    mvn hpi:hpi
+    `mvn clean install`
+
```
Comment (markyjackson-taulia) [inline README.md:25]: Spaces like this are not needed

**Review by markyjackson-taulia — COMMENTED** [review]
I was more looking for new documentation, not grammar fixes

File: README.md (hunk 1 above)
Comment (aaishikasb) [reply README.md:25]: I am working on it. Will it be convenient for you to assign the issue to me?

Comment (markyjackson-taulia) [reply README.md:25]: Because you are not part of the org I cannot assign to you but this is your body of work 🙂

Comment (aaishikasb) [reply README.md:25]: Thanks! I am on it!

**Comment by markyjackson-taulia** [comment]
@aaishikasb Is this in progress?

**Comment by markyjackson-taulia** [comment]
Closing this PR. There has been no progress for 18 days

**Pull request closed without merging**